modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -144,46 +82,79 @@ function AddFDSFilterModalContent({
 	namespace,
 	onSave,
 }: IPropsAddFDSFilterModalContent) {
-	const [selectedField, setSelectedField] = useState<string>();
-	const [label, setLabel] = useState<string>();
-	const [from, setFrom] = useState<string>('');
-	const [to, setTo] = useState<string>('');
+	const fromFormElementId = `${namespace}From`;
```

Comment:
That sounds good to me, I like that structure too because you are sort of defining things as they come within the react tree, context being the highest.

One doubt would be if a context uses something from state or a local variable, but I think that'd be uncommon.